stable/database/values.yaml

```diff
@@ -133,6 +133,10 @@ database:
   # Database password
   rootPassword: secret
 
+  # archiveType
+  # Archive Type (either "" or "lsa")
+  archiveType: ""
```

Comment:
Should we prevent LSA usage if the product version doesn't support it? Otherwise the archiveType doesn't have any effect on the database.
We could detect that and fail fast in nuosm script with an error message.

Reply:
Make sense.